src/parser/jobs/mch/modules/Drift.tsx

```diff
@@ -0,0 +1,95 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import _ from 'lodash'
+import React from 'react'
+
+import ActionLink from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Downtime from 'parser/core/modules/Downtime'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+// Escalate severity depending on how long drill/AA were drifted for
+const DRIFT_SEVERITY_TIERS = {
+	0.5: SEVERITY.MINOR,
```

Comment:
Thinking more on this – since the math works out to the *number* of missing drills/aa, then this will never be a minor (since you call `Math.ceil` to round up). It's either going to be 0 or 1, which pushes it right to a medium.

If that's the intent (medium at 1 miss) then that's fine, but you might want to remove this line or adjust otherwise.

Reply:
It's actually filtering on 'driftedDrills' (the unrounded value), so this shouldn't be an issue.

Reply:
I see – carry on.